ruby/tests/common_tests.rb

```diff
@@ -722,6 +722,16 @@ def test_enum_const_get_helpers
     m = proto_module::TestMessage.new({optional_enum: proto_module::TestEnum::C})
     assert_equal proto_module::TestEnum::C, m.optional_enum_const
     assert_equal proto_module::TestEnum.const_get(:C), m.optional_enum_const
+
+    m = proto_module::TestMessage2.new({foo: 2})
```

Comment:
Foo needs to be an enum field.

Reply:
Added a new test that uses an enum field.